agent/cache/cache.go

```diff
@@ -84,6 +84,7 @@ type typeEntry struct {
 type ResultMeta struct {
 	// Return whether or not the request was a cache hit
 	Hit bool
+	Age time.Duration
```

Comment:
I would add docs to this. It may be as simple as since the value was last loaded into the cache, but clarity that its the age its _in the cache_ vs. _set in the server_ is an important detail. If this is always 0 for background refreshing ones, its important to note that too.